app/scripts/blocker_afd.js

```diff
@@ -0,0 +1,27 @@
+        </div>
+    </div>
+    `
```

Comment:
Could you import the variable / content from `blocker_general` or use `addBlocker()`?